.eslintrc.yml

```diff
@@ -24,10 +24,18 @@ rules:
   # PLUGINS #
   ###########
   # TYPESCRIPT
-  '@typescript-eslint/interface-name-prefix':
+  '@typescript-eslint/naming-convention':
     - error
-    - always
-  '@typescript-eslint/no-angle-bracket-type-assertion': error
+    - selector: interface
+      format:
+        - PascalCase
+      custom:
+        regex: '^I[A-Z]'
+        match: true
```

Comment:
Nice! Much more explicit now!

Reply:
Stolen from the typescript-eslint docs. :wink: